maps.Rmd

```diff
@@ -211,15 +211,15 @@ ggplot(oz_votes) +
   coord_sf(crs = st_crs("+proj=lcc +datum=WGS84"))
 ```
 
-In this map I have used the WGS84 datum (quite reasonably) and Lambert conformal conic (LCC) projection, which is often used in aeronautical applications because straight lines on the map are approximate "great circles" on the globe, and it is generally considered a good projection for regional maps in the middle latitudes. However, the map looks terrible because I haven't set the parameters very wisely. To fix this, I centre the map at longitude 140 and latitude -25, and fix the two standard parallels (lines at which there are no distortions) at latitudes -18 and -36.
+In this example `st_crs()` is given character input in the form of a **proj4string**, though it also accepts numeric **EPSG code** (see <http://www.epsg-registry.org/>). The proj4string in this case indicates that I am using the WGS84 datum and the Lambert conformal conic (LCC) projection. The LCC which is often used in aeronautical applications because straight lines on the map are approximate "great circles" on the globe, and it is generally considered a good projection for regional maps in the middle latitudes. However, the map looks terrible because I haven't set the parameters very wisely. To fix this, I centre the map at longitude 140 and latitude -25, and fix the two standard parallels (lines at which there are no distortions) at latitudes -18 and -36.
```

Comment:
```suggestion
In this example `st_crs()` is given character input in the form of a **proj4 string**, though it also accepts numeric **EPSG code** (see <http://www.epsg-registry.org/>). The proj4string in this case indicates that I am using the WGS84 datum and the Lambert conformal conic (LCC) projection. The LCC which is often used in aeronautical applications because straight lines on the map are approximate "great circles" on the globe, and it is generally considered a good projection for regional maps in the middle latitudes. However, the map looks terrible because I haven't set the parameters very wisely. To fix this, I centre the map at longitude 140 and latitude -25, and fix the two standard parallels (lines at which there are no distortions) at latitudes -18 and -36.
```
?

Is this an R specific term? Googling for it seems to return only R refs on the first page. I presume there must be a more general name?

Reply:
Assuming you're referring to "proj4string"? I think it originates with an old version of PROJ. The impression I have is that officially the preference is now to specify the CRS using well-known text but I have to admit I find the WKT strings for a CRS to be... intimidating!  

https://proj.org/faq.html#what-happened-to-proj-4

My intuition is that people still use the proj4string a lot because it's a best tradeoff between precisions and conciseness in most cases? Anyway, perhaps what I'll do is add a link to proj.org here

Reply:
Added link in 307ce963ab5deae03c0203dabea94c6da4172d83